home/README.md

```diff
@@ -40,7 +40,7 @@ Fidel is a card-linked API that lets developers create web and mobile applicatio
 
 Fidel API runs on top of global payment networks, so it doesn’t require changes to existing merchant infrastructures. No need for new software, POS integrations, staff training or new cards. All the PCI compliance requirements are managed by us so you don’t have to.
 
-Currently, the API is available in the **UK**, **Ireland**, **Sweden**, **Canada** and the **United States**, with **Norway**, **Denmark**, **Finland**, **Japan**, **Australia** and **South Africa** going live soon. Please refer to the up to date list of countries and schemes on [fidel.uk](https://fidel.uk/). We work continuously to add support for other countries and networks. If you would like to deploy card-linked applications in other countries please contact us at [developer@fidel.uk](mailto:developer@fidel.uk).
+Currently, the API is available in the **United States**, **UK**, **Ireland**, **Canada**, **Sweden**, **Norway** and **Finland**, with **Australia**, **Japan** and **South Africa** going live soon. Please refer to the up to date list of countries and schemes on [fidel.uk](https://fidel.uk/). We work continuously to add support for other countries and networks. If you would like to deploy card-linked applications in other countries please contact us at [developer@fidel.uk](mailto:developer@fidel.uk).
```

Comment:
let's change the link to fidel.uk/products, the list of live countries has moved there